app/src/main/kotlin/com/wire/android/util/ImageUtil.kt

```diff
@@ -0,0 +1,76 @@
+package com.wire.android.util
+
+import android.content.Context
+import android.graphics.Bitmap
+import android.graphics.BitmapFactory
+import android.graphics.Matrix
+import android.net.Uri
+import androidx.exifinterface.media.ExifInterface
+import java.io.ByteArrayOutputStream
+
+/**
+ * Rotates the image to a [ExifInterface.ORIENTATION_NORMAL] in case it's rotated with a different orientation
+ *
+ * @param exif Exif interface for of the image to rotate
+ * @return Bitmap the rotated bitmap or the same in case there is no rotation performed
+ */
+private fun Bitmap.rotateImageToNormalOrientation(exif: ExifInterface?): Bitmap {
```

Comment:
what happens when exif is null ?

Reply:
It will do nothing, will return the same bitmap without transformation. I considered this process shouldn't be breaking for the user, but maybe We can get some definitions on what to do if post-processing fails and implement that in a future PR :D